src/components/graph/graph.config.js

```diff
@@ -118,6 +118,13 @@
  * @param {string} [node.highlightFontWeight="normal"] - fontWeight in highlighted state.
  * @param {string} [node.highlightStrokeColor="SAME"] - strokeColor in highlighted state.
  * @param {number} [node.highlightStrokeWidth="SAME"] - strokeWidth in highlighted state.
+ * @param {string} [node.labelPosition=null] - 🔍🔍🔍 location to place node label relative to node.
```

Comment:
```suggestion
 * @param {string} [node.labelPosition=null] - <span id="node-label-position"></span> 🔍🔍🔍 location to place node label relative to node.
```

I added anchors to all the properties to generate more accessible docs. Could you update this accordingly? Thanks